### PR TITLE
Update the age in meta tag descriptions

### LINKED ISSUE #76: Change my age to 14 in the meta tag descriptions
**Is your feature request related to a problem? Please describe.**
Currently, my age on meta tag descriptions on index.html file is still 13 years

**Describe the solution you'd like**
Change my age to 14 in the meta tag descriptions in index.html file

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by Keerthivardhan1**
## Purpose
 hear the issue is to change the age in the index.html files
The purpose of this Pull Request is to fix #76

## Goals
 As per the issue the goal is to update The age in the light/index.html file Facebook OG meta tag description's age 13 years old to 14 years old.
And The age in the Facebook OG meta tag description's age in index.html file is 1 years old change it to 14 years old

## Approach 
step1: clone the repo
step2: read README file and understand code structure of the project .
step3: update the age from 13 to 14 in light/index.html file Facebook OG meta tag.
step4:change the age in  Facebook OG meta tag description's age in index.html file from 1 years old to 14 years old. 
step4: add and commit the changes and push your branch.
  
##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Test environment
Chrome
Version 106.0.5249.119 (Official Build) (64-bit)
 
## Learning
 by understanding the project I learnt how to structure our code and divide in to categories

### CODE REVIEW

File: light/index.html [hunk 1]
```diff
@@ -89,7 +89,7 @@
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
             integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
             crossorigin="anonymous"></script>
-
+     </div>
```
Comment (YoshithaRathnayake) [inline light/index.html:92]: Please remove this unwanted div

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: light/index.html [hunk 2]
```diff
@@ -89,7 +89,7 @@
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
             integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
             crossorigin="anonymous"></script>
-
+     </div>
 </body>
 
 </html>
```
Comment (YoshithaRathnayake) [inline light/index.html:95]: Add a new line at the end of the file. @Keerthivardhan1

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job !
Thank You @Keerthivardhan1

**Comment by Keerthivardhan1** [comment]
Thank you @YoshithaRathnayake  
I am new to git and githhub thanks for barring with me.

**Comment by YoshithaRathnayake** [comment]
> Thank you @YoshithaRathnayake I am new to git and githhub thanks for barring with me.

Any time and you also did well at this issue @Keerthivardhan1

**Comment by YoshithaRathnayake** [comment]
> Thank you @YoshithaRathnayake I am new to git and githhub thanks for barring with me.

Hope you will contribute more in my sites @Keerthivardhan1

**Pull request merged**